theme.js

```diff
@@ -0,0 +1,41 @@
+//Prepares the document (check for jQuery). Remember to load jQuery in the html file BEFORE running any scripts with jQuery.
+$(document).ready(function () {
+
+    // Get the first header iframe element with the id of header
+    var header = $('#header');
+    //get the contents of this id, since this id is connected to an iframe, we want the iframe, not the id of the iframe
+    var headerI = header.contents()[0];
+    //Sets the active page, the html page that is in the URL bar of your browser (since you remember that header and footer are their own html files, but its URL is not loaded in the URL bar)
+    var activePage = window.top.$('html').contents();
+
+    //We get the contents of the footer the same way, since we also want to change its value when we toggle dark/light mode
+    var footer = $('#footer');
```

Comment:
Same thing with the footer, because we want to turn dark mode on the header, main page, and footer, all of which are separate html files.